pants.travis-ci.toml

```diff
@@ -11,6 +11,7 @@ travis_parallelism = 4
 [GLOBAL]
 # Override color support detection - we want colors and Travis supports them.
 colors = true
+dynamic_ui = false
```

Comment:
What the. It seems like maybe travis reports that `stderr` is a TTY, but not `stdout`? Strange.